control/vehicle_cmd_gate/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.5)
+project(vehicle_cmd_gate)
+
+### Compile options
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  # add_compile_options(-Wall -Wextra -Wpedantic)
+  add_compile_options(-Wno-unused-parameter)
+endif()
+
+### Dependencies
+find_package(ament_cmake REQUIRED)
+find_package(autoware_control_msgs REQUIRED)
+find_package(autoware_vehicle_msgs REQUIRED)
+find_package(rclcpp REQUIRED)
+
+# Add path of include dir
+include_directories(include)
+
+# Generate exe file
+add_executable(vehicle_cmd_gate src/vehicle_cmd_gate_node.cpp src/vehicle_cmd_gate.cpp src/vehicle_cmd_filter.cpp)
+
+ament_target_dependencies(vehicle_cmd_gate autoware_control_msgs autoware_vehicle_msgs rclcpp)
+
```

Comment:
I think you also need an `ament_export_dependencies` to make sure upstream packages find the transient dependencies

Reply:
Second that. See https://github.com/ros-planning/navigation2/blob/main/nav2_costmap_2d/CMakeLists.txt#L184-L186.

Reply:
:+1: Done